DailyReportWeb/Controllers/Api/AccountController.cs

```diff
@@ -139,5 +143,26 @@ public void Logout()
         {
             AuthenticationManager.SignOut(DefaultAuthenticationTypes.ApplicationCookie);
         }
+
+        private void SendEmail(ApplicationUser user, string callbackUrl)
```

Comment:
Asta ar trebui sa fie `SendAccountConfirmationEmail`.

Ai putea sa o imparti in 2. o parte in care obtii obiectul `MailMessage` si una in care il trimiti.

`var message = GetAccountConfirmationMessage(user, callbackUrl);
SendEmail(message);`